mx.sulong/testsuites.py

```diff
@@ -22,11 +24,18 @@
 def compileSulongSuite():
     print("Compiling Sulong Suite reference executables ", end='')
     tools.printProgress(tools.multicompileRefFolder(_sulongSuiteDir, _cacheDir, [tools.Tool.CLANG], ['-Iinclude', '-lm']))
-    print("Compiling Sulong Suite with -O0 ", end='')
-    tools.printProgress(tools.multicompileFolder(_sulongSuiteDir, _cacheDir, [tools.Tool.CLANG, tools.Tool.GCC], ['-Iinclude', '-lm'], [tools.Optimization.O0], tools.ProgrammingLanguage.LLVMBC, optimizers=[tools.Tool.BB_VECTORIZE]))
-    print("Compiling Sulong Suite with -O1/2/3 ", end='')
-    tools.printProgress(tools.multicompileFolder(_sulongSuiteDir, _cacheDir, [tools.Tool.CLANG, tools.Tool.GCC], ['-Iinclude', '-lm'], [tools.Optimization.O1, tools.Optimization.O2, tools.Optimization.O3], tools.ProgrammingLanguage.LLVMBC))
-    # MG: compared to the old test suite we do not run the ll files
+    print("Compiling Sulong Suite with clang -O0 ", end='')
```

Comment:
I think we shouldn't hide that information, because it is already rather minimal. Also the progress dots are necessary to workaround the travis timeout.